jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/command/StartCaseCommand.java

```diff
@@ -118,6 +119,10 @@ public void matchCreated(MatchCreatedEvent event) {
         logger.debug("Starting process instance for case {} and case definition {}", caseId, caseDefinitionId);
         CorrelationKey correlationKey = correlationKeyFactory.newCorrelationKey(caseId);
         Map<String, Object> params = new HashMap<>();
+        // add process parent
+        if (caseFile instanceof CaseFileInstanceImpl && ((CaseFileInstanceImpl) caseFile).getParentInstanceId() != null && ((CaseFileInstanceImpl) caseFile).getParentInstanceId() >= 0) {
+            params.put("ParentProcessCaseId", ((CaseFileInstanceImpl) caseFile).getParentInstanceId());
```

Comment:
this key is very weird named - so what it is actually? is it parent process id or parent case id or... you got me confused here

Reply:
@mswiderski ParentInstanceId

Reply:
so would make sense to update it so it is clear what it's meant to be

Reply:
done